hazelcast/src/test/java/com/hazelcast/map/LocalMapStatsTest.java

```diff
@@ -54,6 +54,23 @@ public void testPutAndHitsGenerated() throws Exception {
     }
 
     @Test
+    public void testPutAsync() throws Exception {
+        HazelcastInstance h1 = createHazelcastInstance();
+        IMap<Integer, Integer> map = h1.getMap(randomMapName());
+        for (int i = 0; i < 100; i++) {
+            map.putAsync(i, i);
+        }
+        final LocalMapStats localMapStats = map.getLocalMapStats();
+        assertTrueEventually(new AssertTask() {
+            @Override
+            public void run()
+                    throws Exception {
```

Comment:
I hit the auto format button often :) In most cases there are no changes in the code.